src/style.css

```diff
@@ -1 +1,56 @@
-/* Styles go here */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: 'Roboto', sans-serif;
+  height: 60px;
```

Comment:
don't hardcode `height`, try to use `padding` for list items. check in other places for `height: 60px;`